RecoLocalTracker/SiPixelRecHits/src/PixelCPEFastParamsHost.cc

```diff
@@ -42,19 +42,21 @@ void PixelCPEFastParamsHost<TrackerTraits>::fillParamsForDevice() {
 
   buffer_->commonParams().theThicknessB = m_DetParams.front().theThickness;
   buffer_->commonParams().theThicknessE = m_DetParams.back().theThickness;
-  buffer_->commonParams().thePitchX = m_DetParams[0].thePitchX;
-  buffer_->commonParams().thePitchY = m_DetParams[0].thePitchY;
+  // buffer_->commonParams().thePitchX = m_DetParams[0].thePitchX;
+  // buffer_->commonParams().thePitchY = m_DetParams[0].thePitchY;
```

Comment:
could you remove the commented-out code (here an below) ?

Reply:
Yep